### ISSUE TITLE
Github Notebook: issue icon based on themes

### ISSUE THREAD

**Issue by rebornix** [microsoft/vscode#93986]
Issue Type: <b>Bug</b>

Currently the issue icon is dark in dark theme.

<img width="2098" alt="image" src="https://user-images.githubusercontent.com/876920/78050949-d9a0b080-7331-11ea-970b-6bf07f1bf59d.png">

VS Code version: Code - OSS Dev 1.44.0 (Commit unknown, Date unknown)
OS version: Darwin x64 19.2.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i9-9980HK CPU @ 2.40GHz (16 x 2400)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off_ok<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 3, 3|
|Memory (System)|32.00GB (2.60GB free)|
|Process Argv|. --no-sandbox --extensions-dir /Users/penlv/code/work/notebook-extension-samples|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (3)</summary>

Extension|Author (truncated)|Version
---|---|---
notebook-test|vsc|1.0.0
notebook-regexper|vsc|1.0.0
notebook-renderers|vsc|1.0.0

</details>

**Comment by jrieken** [comment]
Yes - this need love. I literally copied the SVG-source from their website...
